source/electrical-power-systems/index.md

```diff
@@ -0,0 +1,46 @@
+Such power systems can be broken down into overhead and "third rail" systems.
+
+## Overhead Catenary Wire
+
+Catenary wire is strung up over the tracks.
+
+Rolling stock which needs power is connected to the overhead wire with a *pantograph*.
+
+Modern overhead wire systems for mainline rail are typically at higher voltages AC, such as 25 kV.
+
+## Third Rail
+
+A "third rail" was originally just that, a third rail not used to support the train, but instead power it.
+(Modern third rail has diverged in material and shape from regular rail, but the name has stuck.)
+
+The third rail is much thicker than a regular wire, and as such can carry higher currents.
+At the same time, being right at ground level, and less well insulated, the third rail requires lower voltage.
+Lower voltage but higher current roughly cancels out in terms of power (their product), however.
+
+### Inferiority
+
+The physical power delivery mechanism itself are much more expensive for third rail.
+Third rails only last about 20 years.[^third-rail-bart]
+Overhead wire itself and insulators, by contrast, typically lasts over twice as long as that[^overhead-wire-longevity],
+and the support poles often last even longer at over 100 years.
+Together, these costs and lifespans make the amortized costs of overhead wire construction and maintenance far lower.
+
+[^third-rail-bart]: <https://www.bart.gov/news/articles/2019/news20190731>
+
+[^overhead-wire-longevity]: <https://www.sdtools.com/pdf/WCRR11_fatigue.pdf>
+
+Third rail also requires the amount of supporting infrastructure needed.
+Because a third rail can only handle low voltages, each substation can only feed a few miles of track.
+By contrast, a substation feeding high-voltage AC to an overhead wire can instead cover tens of route miles each.
+More frequent substations and associated transmission (connecting the right-of-way to the substations, and the substations to the grid) drives the cost of new third rail higher than new overhead wire.
+
+Other maintenance beyond the power system becomes much simpler, easier, and safer, with overhead catenary wire – the efficiency gains from better power systems extend beyond the power systems itself.
+With third rail, just about any track-level maintenance requires cutting power to the third rail if present.
+By contrast, since overhead wire is out of the way of both equipment of workers, it can safely be kept powered on during many maintenance operations.
+Modern maintenance of way (MOW) equipment can have height limiters installed to avoid strikes or arcing, safely enabling tasks like replacing track components like bolts, tie spikes, or clips without deenergization.
+For many ground-level track renewal tasks (tie, ballast, etc.), the third rails (and covers) are physically in the way and have to be removed and reinstalled for the work to go ahead, adding significant time and expense.
+By contrast, overhead wire is not in the way and can be left in place.
```

Comment:
This bit was taken from a draft report, so is hopefully more correct, but do double check :)